includes/common.php

```diff
@@ -1,19 +1,33 @@
 <?php
 /**
  * Function to get donation and original price out of an order.
+ * 
+ * @param object $order The order object.
+ * @since TBD
+ * return array The price components.
  */
 function pmprodon_get_price_components( $order ) {
 	$r = array(
 		'price'    => $order->total,
 		'donation' => '',
 	);
 
-	if ( isset( $order->notes ) && ! empty( $order->notes ) && strpos( $order->notes, __( 'Donation', 'pmpro-donations' ) ) !== false ) {
-		$donation      = pmpro_getMatches( '/' . __( 'Donation', 'pmpro-donations' ) . '\: ([0-9\.]+)/', $order->notes, true );
-		$r['donation'] = $donation;
-		if ( $donation > 0 ) {
-			$r['price'] = $order->total - $donation;
-		}
+	global $wpdb;
+	$table_name = $wpdb->pmpro_membership_ordermeta;
+	$donation = 0;
+	// check for donation in order meta table.
+	if ( $wpdb->get_var( $wpdb->prepare( "SHOW TABLES LIKE %s", $table_name ) ) === $table_name ) {

```

Comment:
The ordermeta table has been around for over 3 years at this point. We shouldn't need to check that it exists anymore, we can just call `get_pmpro_membership_order_meta()`